blocks-config/uagb-controls/AddGBSStyles.js

```diff
@@ -35,8 +35,15 @@ const AddGBSStyles = ( ChildComponent )=> {
 		} ), [ globalBlockStyleId, globalBlockStyles ] );
 
 		useEffect( () => {
-			const editorStyles = getGBSEditorStyles( globalBlockStyles, globalBlockStyleId );
-			AddGBSStylesDom( globalBlockStyleId, editorStyles );
+			if ( uagb_blocks_info?.spectra_pro_status && 'enabled' === uagb_blocks_info?.uag_enable_gbs_extension ) {
+				const editorStyles = getGBSEditorStyles( globalBlockStyles, globalBlockStyleId );
+				AddGBSStylesDom( globalBlockStyleId, editorStyles );
+			}
+
+			// Don't set attribute is extension is not enabled.
+			if( 'disabled' === uagb_blocks_info?.uag_enable_gbs_extension ){
```

Comment:
Minor Coding Standard Improvement - Check in this file where possible.

```suggestion
			if ( 'disabled' === uagb_blocks_info?.uag_enable_gbs_extension ){
```

Reply:
Done